plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/statistics/DeltaLakeJsonFileStatistics.java

```diff
@@ -143,6 +150,15 @@ private Optional<Object> deserializeStatisticsValue(DeltaLakeColumnHandle column
                 return Optional.empty();
             }
         }
+        if (columnType instanceof TimestampType) {
+            long epochMicros = (long) columnValue;
+            long epochSeconds = floorDiv(epochMicros, MICROSECONDS_PER_SECOND);
+            int nanoAdjustment = floorMod(epochMicros, MICROSECONDS_PER_SECOND) * NANOSECONDS_PER_MICROSECOND;
+            LocalDateTime localDateTime = LocalDateTime.ofEpochSecond(epochSeconds, nanoAdjustment, UTC);
+            if (localDateTime.toLocalDate().isBefore(START_OF_MODERN_ERA)) {
+                return Optional.empty();
+            }
```

Comment:
unnecessary calculations

this could be written simply as 
```
if (epochMicros < START_OF_MODERN_ERA_EPOCH_MICROS) {
                return Optional.empty();
            }
```

Reply:
https://github.com/trinodb/trino/pull/18125 updates existing code.